java-components/management-console/src/main/java/com/redhat/hacbs/management/model/ContainerImage.java

```diff
@@ -58,7 +60,45 @@ public static ContainerImage findImage(String image) {
         return containerImage;
     }
 
+    public static ContainerImage get(String image) {
+        return get(null, image);
+    }
+
+    public static ContainerImage get(ContainerImageRepository repository, String image) {
```

Comment:
@stuartwdouglas Might be worth speaking about the logic for image lookups and what formats should be acceptable.

Reply:
In the DB you really need the digest, unless you are happy to get multiple results back. Tags can change so digest is the only really unique way to identify the image.

Reply:
Yeah. Its not obvious on addImage in the UI what formats are acceptable. Also, if I'm using jib to push an image/tag to Quay I don't know the digest (not sure if I can get jib to tell me the digest of the image it just pushed)

Reply:
You can get the digest of the image, it is included in the pipeline results.